src/plugins/embeddable/public/lib/attribute_service/attribute_service.tsx

```diff
@@ -86,8 +86,12 @@ export class AttributeService<
   public async unwrapAttributes(input: RefType | ValType): Promise<SavedObjectAttributes> {
     if (this.inputIsRefType(input)) {
       return this.options.unwrapMethod
-        ? await this.options.unwrapMethod(input.savedObjectId)
-        : await this.defaultUnwrapMethod(input);
+        ? await this.options.unwrapMethod(input.savedObjectId).catch((e: Error) => {
```

Comment:
Is there a reason for adding the catches if they are just rethrowing?

Reply:
Good catch! Cleaned all of those up, thanks